parser.go

```diff
@@ -15,11 +15,12 @@ import (
 // should be deterministically ordered.
 type UDPMetric struct {
 	MetricKey
-	Digest     uint32
-	Value      interface{}
-	SampleRate float32
-	Tags       []string
-	LocalOnly  bool
+	Digest      uint32
+	Value       interface{}
+	SampleRate  float32
+	Tags        []string
+	LocalOnly   bool
+	ForceGlobal bool
```

Comment:
Aren't `LocalOnly` and `ForceGlobal` always mutually exclusive? I wonder if we can combine this, maybe into an enum:

e.g. 

```go
type UDPMetric struct{
Scope MetricScope
}

type MetricScope int

const (
MixedScope MetricScope = iota
LocalOnly
GlobalOnly
)

```